egs2/chime7_task1/asr1/README.md

```diff
@@ -158,69 +164,194 @@ chime7_task1
+    <td rowspan="3">30.8</td>
+  </tr>
+  <tr>
+    <td>DiPCo</td>
+    <td>GSS (EV top 80%)</td>
+    <td>36.8</td>
+  </tr>
+  <tr>
+    <td>Mixer-6</td>
+    <td>GSS (EV top 80%)</td>
+    <td>21.2</td>
+  </tr>
+</tbody>
+</table>
+
+Such baseline system would rank four on dev set based on the rules of the past CHiME-6 Challenge 
+on Track 1 (unconstrained LM). 
+The system with top 80% mics and EV selection will be used as the baseline also for evaluation.  
+
+
+Here we present results when only the outer microphones are used on CHiME-6 and DiPCO, while Mixer 6 uses all microphones. <br> 
+**However such system will be against the rules set for this Challenge as such 
+manual microphone selection based on the domain accounts for domain identification.** 
+It is however worth to report these results. <br>
+Here all outer for DiPCo means we used channels `2,5,9,12,16,19,23,26,30,33` (diametrically opposite mics on each array).
+<table>
+<thead>
+  <tr>
+    <th>Dataset</th>
+    <th>split</th>
+    <th>front-end</th>
+    <th>SA-WER (%)</th>
+    <th>macro SA-WER (%)</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>CHiME-6</td>
+    <td rowspan="3">dev</td>
+    <td>GSS (all outer)</td>
+    <td>35.5</td>
+    <td rowspan="3">31.8</td>
+  </tr>
+  <tr>
+    <td>DiPCo</td>
+    <td>GSS (all outer)</td>
+    <td>39.3</td>
+  </tr>
+  <tr>
+    <td>Mixer-6</td>
+    <td>GSS (all)</td>
+    <td>20.6</td>
+  </tr>
+</tbody>
+</table>
+
+As seen it seems that for Mixer 6 Speech using all microphones will be better, but this is not true for CHiME-6 and DiPCo as seen by the channel 
+selection results. <br>
+One of our hopes is that participants can devise new techniques to improve in such regard and devise
+better channel selection/fusion strategies. 
+
+## <a id="eval_script"> 4. Evaluation Script </a>
+
+Will be added together with the diarization baseline. 
+Evaluation will be performed as described in the [Task Main Page](https://www.chimechallenge.org/current/task1/index) and needs joint diarization and transcription. 
+by computing SA-WER for each speaker, where the hypothesis are re-ordered based on the best reordering defined by diarization error rate.
 
 
 ## <a id="common_issues"> 5. Common Issues </a>
 
 1. `AssertionError: Torch not compiled with CUDA enabled` <br> for some reason you installed Pytorch without CUDA support. <br>
  Please install Pytorch with CUDA support as explained in [pytorch website](https://pytorch.org/).
+2. `ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: 'YOUR_PATH/espnet/tools/venv/lib/pyth
+on3.9/site-packages/numpy-1.23.5.dist-info/METADATA'`. This is due to numpy installation getting corrupted for some reason.
+You can remove the site-packages/numpy- folder manually and try to reinstall numpy 1.23.5 with pip. 
 
+## Acknowledgements
 
-
-[sox]:
-[google_group]: 
-[gpu_gss]:
-[gss]: 
-
+We would like to thank Dr. Naoyuki Kamo for his precious help. 
```

Comment:
Thank you for mention me, but sorry, I don't have  Ph.D. :->

Reply:
Oh I am sorry. I can remove it. Honestly you deserve an honorary one ;)